src/common/services/Constants.ts

```diff
@@ -17,6 +17,7 @@ export const BAP_API_VERSION = '2021-04-01';
 export const BAP_SERVICE_SCOPE_DEFAULT = "https://api.bap.microsoft.com/.default";
 export const BAP_SERVICE_ENDPOINT = `{rootURL}/providers/Microsoft.BusinessAppPlatform/`;
 export const BAP_SERVICE_COPILOT_CROSS_GEO_FLAG_RELATIVE_URL = `scopes/admin/environments/{environmentID}?$expand=properties/copilotPolicies&api-version={apiVersion}`;
+export const BAP_ENVIRONMENT_LIST_URL = `scopes/admin/environments?api-version=2021-04-01&select=name,properties.displayName,properties.linkedEnvironmentMetadata`;
```

Comment:
@amitjoshi438 could you please double check the URL? I am a bit worried about the date. How many api versions are supported at a time? How would we know if a version is being deprecated.

Reply:
Created a tracking item for following up on this.